src/main/clojure/clara/rules/logger.cljc

```diff
@@ -0,0 +1,25 @@
+(ns clara.rules.logger)
```

Comment:
If we go with this approach, I'd not call the ns "logger".

Something like `analyzer` or `analysis` seems more typical. Logging is a separate concern.